include/fast_float/ascii_number.h

```diff
@@ -10,7 +10,7 @@
 
 namespace fast_float {
 
-fastfloat_really_inline bool is_integer(char c)  noexcept  { return (c >= '0' && c <= '9'); }
+fastfloat_really_inline bool is_integer(char c)  noexcept  { return (c & 0x30) == 0x30; }
```

Comment:
@lemire  Related to (c & 0x30) == 0x30, I first tried to benchmark this variant with https://github.com/lemire/simple_fastfloat_benchmark, it does not make any performance improvements on my machine, but I checked on godbolt.org only with -O2 optimizations gcc, clang produces the same code. This can be reverted.